contracts/cw721-base/src/execute.rs

```diff
@@ -127,6 +141,60 @@ where
         let ownership = cw_ownable::update_ownership(deps, &env.block, &info.sender, action)?;
         Ok(Response::new().add_attributes(ownership.into_attributes()))
     }
+
+    pub fn set_withdraw_address(
+        &self,
+        deps: DepsMut,
+        sender: &Addr,
+        address: String,
+    ) -> Result<Response<C>, ContractError> {
+        cw_ownable::assert_owner(deps.storage, sender)?;
+        deps.api.addr_validate(&address)?;
+        self.withdraw_address.save(deps.storage, &address)?;
+        Ok(Response::new()
+            .add_attribute("action", "set_withdraw_address")
+            .add_attribute("address", address))
+    }
+
+    pub fn remove_withdraw_address(
+        &self,
+        storage: &mut dyn Storage,
+        sender: &Addr,
+    ) -> Result<Response<C>, ContractError> {
+        cw_ownable::assert_owner(storage, sender)?;
+        let address = self.withdraw_address.may_load(storage)?;
+        match address {
+            Some(address) => {
+                self.withdraw_address.remove(storage);
+                Ok(Response::new()
+                    .add_attribute("action", "remove_withdraw_address")
+                    .add_attribute("address", address))
+            }
+            None => Err(ContractError::NoWithdrawAddress {}),
+        }
+    }
+
+    pub fn withdraw_funds(
+        &self,
+        storage: &mut dyn Storage,
+        amount: &Coin,
+    ) -> Result<Response<C>, ContractError> {
+        let address = self.withdraw_address.may_load(storage)?;
+        match address {
+            Some(address) => {
+                let msg = BankMsg::Send {
+                    to_address: address,
+                    amount: vec![amount.clone()],
+                };
+                Ok(Response::new()
+                    .add_message(msg)
+                    .add_attribute("action", "withdraw_funds")
+                    .add_attribute("amount", amount.amount.to_string())
+                    .add_attribute("denom", amount.denom.to_string()))
+            }
+            None => Err(ContractError::NoWithdrawAddress {}),
+        }
+    }
```

Comment:
Any(!) can withdraw funds - but funds send to given address provided by owner.
Imo allowing any is more flexible, like allowing a team to withdraw funds. Same principle like for fee share module.